.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
This was changed by VSCode automatically: https://stackoverflow.com/questions/77637621/vscode-workspace-settings-change-on-its-own